Makefile

```diff
@@ -67,6 +67,21 @@ publish_to_maven_local:
 publish_to_artifactory:
 	./gradlew -p subprojects publishToArtifactory -PprojectVersion=$(version) $(log_level)
 
+# precondition:
+# - installed CLI: https://cli.github.com/
+# - push $(version) branch
+#
+# post actions:
+# - go to link in output
+# - edit notes
+# - publish release
+draft_release:
+	gh release create $(version) \
```

Comment:
I'd like to have more fail-fast check for preconditions.
At least for variables.

```bash
@[ "${version}" ] || ( echo ">> version is not set"; exit 1 )
```
https://stackoverflow.com/questions/10858261/abort-makefile-if-variable-not-set

Reply:
Thanks, didn't know how to properly add it; let's fix

Reply:
done for vars